h2o-dist/index.html

```diff
@@ -817,6 +821,8 @@ <h2>User Documentation</h2>
           <div id="user_documentation"></div>
           <h2>Developer Documentation</h2>
           <div id="developer_documentation"></div>
+          <h2>Security Documentation</h2>
+          <p><a href="https://docs.h2o.ai/h2o/latest-stable/h2o-docs/security.html" target="_blank"> Security Guidelines</a></p>
```

Comment:
minor - it should point to release specific security guidelines.

Reply:
Perhaps I am missing something, but I don't believe there's security entirely specific to just releases here? Seems like different types of initialization, but I could be misunderstanding. Is there a specific section you have in mind?

Reply:
There might be a minor changes in h2o documentation for older h2o versions. So if you share the download page for an older h2o. (for this and subsequent versions in future). The documentation doesn't have to be precisely accurate.